CHANGELOG.md

```diff
@@ -197,7 +217,9 @@
 - DES-1687: Add rollingback to worker tasks (#30)
 - DES-1663: Fix image rotations (#36)
 
-[unreleased]: https://github.com/TACC-Cloud/geoapi/compare/v2.17...HEAD
+[unreleased]: https://github.com/TACC-Cloud/geoapi/compare/v2.19...HEAD
```

Comment:
repeat comment from other PR: Why are we listing unreleased branches here? If it is an unreleased WIP, I don't see why it would be added to the change log. Am I missing something?